Source/Forms/Gen5/IDs5.cpp

```diff
@@ -115,6 +116,12 @@ void IDs5::setupModels()
     setting.endGroup();
 }
 
+void IDs5::setXOR(bool checked)
+{
+    ui->checkBoxXOR->setChecked(false);
```

Comment:
I don't think this line is necessary

Reply:
I added it to prevent this from happening: 
![image](https://user-images.githubusercontent.com/26980015/135294240-9f4c97af-cc14-4d8e-b9bd-0e1b484457fe.png)

If both PID and XOR are checked, then PID is unchecked, the XOR checkbox will still be checked. Is there another way to prevent this from happening?